crates/turborepo/src/main.rs

```diff
@@ -66,6 +71,8 @@ fn run_go_binary(execution_state: ExecutionState) -> Result<i32> {
 // This function should not expanded. Please add any logic to
 // `turborepo_lib::main` instead
 fn main() -> Result<()> {
+    std::panic::set_hook(Box::new(panic_handler));
```

Comment:
Just checking, does `set_hook` propagate to all threads?

Reply:
Yeah, seems like it does. The docs mention that the hook gets called when "a thread panics", so I assume that's any thread